python-avd/schema_tools/metaschema/resolvemodel.py

```diff
@@ -27,13 +27,15 @@ def merge_schema_from_ref(schema: dict) -> dict:
     ref_schema = merge_schema_from_ref(get_schema_from_ref(ref))
     if ref_schema["type"] != schema["type"]:
         # TODO: Consider if this should be a pyavd specific error
-        raise ValueError(
+        msg = (
             f"Incompatible schema types from ref '{ref}' ref type '{ref_schema['type']}' schema type '{schema['type']}'\nschema: {schema}\nref_schema:"
             f" {ref_schema})"
         )
+        raise ValueError(
+            msg,
+        )
```

Comment:
```suggestion
        raise ValueError(msg)
```

Reply:
All these were caused by an autofix in ruff...